### DISCUSSION TITLE
Regarding the Installation of Tractus-X EDC on my system.

### DISCUSSION THREAD

**Discussion by junaidtk** [eclipse-tractusx/tractusx-edc#334] [General]
Hi 

I have to run the Tractus-X EDC on my system. I am trying it for testing the catenax connection with Tractus-X EDC. I would be much grate full, if anyone can give exact step by step details of running the Tractus-X EDC.

Currently I have just cloned git repository of Tractus-X EDC project to my system.

I have no any Catenax account.

My aim is to run Tractus-X EDC and do some test data exchange between Catenax using Tractus-X EDC for understanding the working.

I also need some test/demo account details for testing the Catenax and Tractus-X EDC connection.

Do you have a common test account ? Or Do I need to create a new Catenax account ?

Thanking in advance.

**Comment by Bailonis** [comment]
Hi,

I'm in the same situation, do you had any updates?

Thanking in advance.

**Reply by rolfwessels** [reply]
I'm also in the same situation. Any help would be appreciated 🙇

**Comment by stephanbcbauer** [comment]
Hey @Bailonis , @junaidtk 

Thank you very much for starting this discussion, :) The Catnea-x account is only needed if you want to communicate with other tractusx-edcs within the catnea-x network. Because you have to register your EDC with your portal account.

If you want to test the EDC locally you can "deploy" two of them locally.

For me, it's really important to get feedback about current documentation because we want to improve it. So what did you do until now? Which documentation was your entry point? My recommendation would be the tractus X developer hub https://eclipse-tractusx.github.io/docs/kits/tractusx-edc/docs/kit/adoption-view/Adoption%20View

The section operation view should help you both. And if not, please give feedback. :)

thx

**Comment by Bailonis** [comment]
I have been following the documentation from here: https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-controlplane/edc-runtime-memory
From I read this is to make a local test, that is what i need at the moment, in future, I will start with the portal.

I follow all the stpes, when when it it runs, it gives me error  in this part:

![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/72702458/5098b266-4e7e-45b0-91fa-4ab3f3a86f94)

![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/72702458/d3443588-5f3d-4ee4-b7d4-242bd82ba961)

I'm using this:

edc.oauth.token.url=https://daps1.int.demo.catena-x.net/token
edc.oauth.provider.jwks.url=https://daps1.int.demo.catena-x.net/jwks.json

**Comment by Bailonis** [comment]
@stephanbcbauer For now I'm trying to do just a local test, the portal would be in a near future.

I have been reading the documentation from here: https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-controlplane/edc-runtime-memory

but having problems in this part:
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/72702458/e412e3fb-7b7b-43cf-b4b4-7ef7869803d1)

I have been using this:
edc.oauth.provider.jwks.url=https://daps1.int.demo.catena-x.net/jwks.json
edc.oauth.token.url=https://daps1.int.demo.catena-x.net/token

but it gaves me this error:
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/72702458/7fb86e87-e939-4212-b4e5-a4a6fc1da77c)

Do I need to be registred into the Portal to make a local Test??

**Comment by stephanbcbauer** [comment]
@Bailonis if you want to test the connection and also get data from a specific catena-x environment (with your current configuration it would be the INT environment -> not sure if this is correct for you) than yes you need a catena-x account and also register your EDC within the portal. -> Your company has propably already a catena x account? Maybe to beta environment?

If you just want to test the tractusx-edc locally you will need two edcs (consumer/provider -> i think we call it socrates/plato) -> they can communicate and also share data

**Comment by stephanbcbauer** [comment]
And maybe additional information. We are working on a step by step guide (its not merged yet) -> but it would be really cool if you could test it (in case it fits to your use case) and also provide feedback for it. thx #314

**Comment by stephanbcbauer** [comment]
@Bailonis , @junaidtk what the current state about your tests? Does it work? did the documentation help? If yes, which one? THX

**Reply by Bailonis** [reply]
Hi there!

I successfully run the sample [local setup](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/example-dataspace/README.md). with plato and sokrates!

Thanks for the help!

**Reply by stephanbcbauer** [reply]
@Bailonis congrats. Would you also be able to support a little bit? @junaidtk

**Comment by junaidtk** [comment]
@stephanbcbauer 
I have checked the link (https://eclipse-tractusx.github.io/docs/kits/tractusx-edc/docs/kit/adoption-view/Adoption%20View) you shared on your first comment. But it is redirecting to a page not found page.

Regarding the test. I successfully run the sample [Local setup](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/example-dataspace/README.md). Which is one that I need to next for familiarisation ?

But I still have some confusion, from where I have to start and How can i do a test transfer ?

Regarding the ticket [314](https://github.com/eclipse-tractusx/tractusx-edc/pull/314). Which is the link we have to follow, It would be much appreciated , if you can just list here the links that we need to follow.

Do we have any test account for the Catena-x ?

**Reply by stephanbcbauer** [reply]
@junaidtk we created a new [example](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/example-dataspace/README.md) Maybe this already helps?

And sorry for the Connector Kit Link. They changed the hierarchy a little bit :) https://eclipse-tractusx.github.io/docs-kits/kits/tractusx-edc/CHANGELOG

**Comment by stephanbcbauer** [comment]
@Bailonis @junaidtk can you please sync and help each other? Did it work?

**Reply by Bailonis** [reply]
Yes, of course, any doubt that @junaidtk have I can try to help :)

**Comment by stephanbcbauer** [comment]
@Bailonis what was the information, link, documentation which helped you in the end? Thx

**Reply by Bailonis** [reply]
This one: https://github.com/eclipse-edc/Samples/

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
